### PR TITLE
Master profile data

### PR DESCRIPTION
**Pull request by ennaque**

### CODE REVIEW

File: src/app/master/components/certificate/certificate.component.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.certificate {
```
Comment (enkryptor) [inline src/app/master/components/certificate/certificate.component.scss:1]: We really should use standard common styles, not custom ones

File: src/app/master/components/certificate-edit/certificate-edit.component.html [hunk 2]
```diff
@@ -60,14 +60,26 @@
...
-                </ion-button>
-
-                <ion-button (click)="save()" expand="block">
-                    {{"certificates-edit-page.save"|translate}}
-                </ion-button>
+                <ion-row class="ion-justify-content-center ion-margin-top">
+                    <ion-col size="6">
+                        <ion-button
+                                (click)="delete()"
+                                *ngIf="item.id"
```
Comment (enkryptor) [inline src/app/master/components/certificate-edit/certificate-edit.component.html:67]: A user can't delete a non-saved item? Why?

File: src/app/master/components/certificate/certificate.component.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <ion-buttons *ngIf="editable">
+        <ion-button size="medium" routerDirection="forward"
+                    routerLink="certificate-edit/{{certificate.id}}"
+                    routerLinkActive="active">
+            <ion-icon name="create-outline"></ion-icon>
+        </ion-button>
+    </ion-buttons>
+</ion-row>
+<ion-label color="medium">{{certificate.date}}</ion-label>
+<ion-img class="certificate" *ngIf="certificate.photo" [src]="sanitizer.sanitize(certificate.photo)"></ion-img>
```
Comment (enkryptor) [inline src/app/master/components/certificate/certificate.component.html:12]: `[src]="sanitizer.sanitize()` — We shouldn't transform data in a presentation layer.

The whole `sanitize()` thing is probably unneeded.

File: src/app/master/components/certificate/certificate.component.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+  padding: 7px 14px 7px 7px;
```
Comment (enkryptor) [inline src/app/master/components/certificate/certificate.component.scss:2]: What is "7px"? Is it really very specific and unique paddings, or can we have a generic style for them?

**Review by enkryptor — COMMENTED** [review]
We have to deal with the sanitizer problem first.

**Review by enkryptor — CHANGES_REQUESTED** [review]
Replace `[src]="sanitizer.sanitize(certificate.photo)"` with `[src]="certificate.photo"`, remove the sanitizer dependency. Everything else is fine.

File: src/app/master/components/certificate/certificate.component.html (hunk 3 above)
Comment (ennaque) [reply src/app/master/components/certificate/certificate.component.html:12]: later

**Pull request merged**